### PR TITLE
Demo linters in ci pipeline

### PR DESCRIPTION
**Pull request by epavlova**

### CODE REVIEW

File: annotations/annotations_api.go [hunk 1]
```diff
@@ -36,7 +36,12 @@ const (
 	nextVideoAnnotationLifecycle = "next-video"
 )
 
-// UPPError encapsulate error information for errors originating from calls to UPP annotations endpoint.
+type UPP1Error struct {
+	msg     string
```
Comment (epavlova) [inline annotations/annotations_api.go]: **[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
`msg` is unused (structcheck)

File: annotations/annotations_api.go [hunk 2]
```diff
@@ -36,7 +36,12 @@ const (
...
+	status  int
```
Comment (epavlova) [inline annotations/annotations_api.go]: **[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
`status` is unused (structcheck)

File: annotations/annotations_api.go [hunk 3]
```diff
@@ -36,7 +36,12 @@ const (
...
+	uppBody []byte
```
Comment (epavlova) [inline annotations/annotations_api.go]: **[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
`uppBody` is unused (structcheck)

**Comment by karsov** [comment]
Not needed now.

**Pull request closed without merging**